pkg/cloud/aws/services/userdata/node.go

```diff
@@ -37,9 +37,9 @@ HOSTNAME="$(curl http://169.254.169.254/latest/meta-data/local-hostname)"
 cat >/tmp/kubeadm-node.yaml <<EOF
 ---
 apiVersion: kubeadm.k8s.io/v1alpha3
-kind: NodeConfiguration
+kind: JoinConfiguration
```

Comment:
How is this different than before? Is there any upside to it?

Reply:
NodeConfiguration is not a valid type

Reply:
How would it work if this isn't correct? :/

Reply:
Interesting, I think NodeConfiguration might have been only available in v1alpha2